portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* Creates Slideshow Carousel to go through my projects*/
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+var slideIndex = 1;
+showSlides(slideIndex += n);
```

Comment:
Sorry I am new to JS so my question could be dump. Why is this needed?

Reply:
no you're right!

Reply:
Done